ulib/fs/VS/CallProc/CallProc.fsproj

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>25b60dd4-fbf7-431b-9f6b-2ef45a7306c9</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>CallProc</RootNamespace>
+    <AssemblyName>CallProc</AssemblyName>
+    <OtherFlags>--nowarn:0086 --mlcompatibility --nologo</OtherFlags>
```

Comment:
This is important! Otherwise, if the environment sets the OTHERFLAGS variable (which our build scripts routinely do) then the value of that variable is passed to `fsc`. So, you must set <OtherFlags> here. Crazy, but true.